ores/wsgi/routes/v2.py

```diff
@@ -0,0 +1,178 @@
+    @bp.route("/v2/scores/<context>/", methods=["GET"])
+    def score_model_revisions_v2(context):
+
+        # Check to see if we have the context available in our score_processor
+        if context not in score_processor:
+            return responses.not_found("No scorers available for {0}"
+                                       .format(context))
+
+        # If no model is specified, return information on available models
+        if "models" not in request.args:
+            # Return the models that we have
+            models = {name: model.info()
+                      for name, model in score_processor[context].items()}
+            return jsonify({"models": models})
+
+        # Read the params
+        try:
+            models = set(read_bar_split_param(request, "models", type=str))
+        except ParamError as e:
+            return responses.bad_request(str(e))
+
+        # Get model info
+        try:
+            model_info_req = set(read_bar_split_param(request, "model_info", type=str))
+        except ParamError as e:
+            return responses.bad_request(str(e))
+
+        # Check if all the models are available
+        missing_models = models - score_processor[context].keys()
+        if len(missing_models) > 0:
+            return responses.bad_request("Models '{0}' not available for {1}."
+                                         .format(list(missing_models), context))
+
+        # Read the rev_ids
+        try:
+            rev_ids = set(read_bar_split_param(request, "revids", type=int))
+        except ParamError as e:
+            return responses.bad_request(str(e))
+
+        if len(rev_ids) == 0:
+            return responses.bad_request("No revids provided.")
+
+        precache = "precache" in request.args
+
+        # Generate scores for each model and merge them together
+        scores = defaultdict(dict)
+        model_info = {}
+        try:
+            for model in models:
+                model_object = score_processor[context][model]
+                model_info[model] = {'version': model_object.version}
+                if model_info_req:
+                    try:
+                        for req in model_info_req:
+                            model_info[model][req] = model_object.info()[req]
+                    except KeyError:
+                        return responses.bad_request(
+                            "Model '{0}' has not attribute {1}.".format(
+                                model, req))
+                model_scores = score_processor.score(
+                    context, model, rev_ids, precache=precache)
+                scores[model] = model_scores
+        except errors.ScoreProcessorOverloaded:
+            return responses.server_overloaded()
+        return format_output(context, scores, model_info)
+
+    # /v2/scores/enwiki/reverted/?revids=456789|4567890
+    @bp.route("/v2/scores/<context>/<model>/", methods=["GET"])
+    def score_revisions_v2(context, model):
```

Comment:
Should have score_revisions_v2 and score_revision_v2 share code.  I imagine a function that takes a `context`, `models` and `rev_ids` and produces the same structure dictionary.